### PR TITLE
SV-444 Added postman test collection for standards endpoint in all apis

### PR DESCRIPTION
**Pull request by DevenKShah**
Begins with querying Get List on courses api and stores data in the environment which is used to assert calls to outer apis. 

The tests are written so that the counts match as well. Currently the filters are applied as such: 
Approvals = `ActiveAvailable`
Assessor = `Active`
Campaign = N/A as Campaign gets Lars Codes by sector
FAT = `ActiveAvailable`
Find EPAO = `Active`
Forecasting = `ActiveAvailable`
MA = `Active`
Recruit = Cannot tell as the same response brings back frameworks and standards
Reservations = `ActiveAvailable`

### CODE REVIEW

**Review by alandburns — APPROVED** [review]

File: IntegrationTests/Outer API Tests For Standards.postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			"enabled": true
+		},
+		{
+			"key": "courses-api-client-secret",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "courses-api-resource",
+			"value": "https://citizenazuresfabisgov.onmicrosoft.com/das-test2-crsapi-as-ar",
```
Comment (dashton82) [inline IntegrationTests/Outer API Tests For Standards.postman_environment.json:22]: this kind of information should not be in this repo

**Review by dashton82 — CHANGES_REQUESTED** [review]

File: IntegrationTests/Outer API Tests For Standards.postman_environment.json (hunk 1 above)
Comment (DevenKShah) [reply IntegrationTests/Outer API Tests For Standards.postman_environment.json]: Removed

File: IntegrationTests/Outer API Tests For Standards.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,765 @@
...
+					"mode": "urlencoded",
+					"urlencoded": [
+						{
+							"key": "grant_type",
+							"value": "client_credentials",
+							"type": "text"
+						},
+						{
+							"key": "client_id",
+							"value": "e94608b1-0e4c-45dc-aecd-1a8442e9a58e",
```
Comment (dashton82) [inline IntegrationTests/Outer API Tests For Standards.postman_collection.json:50]: 😱

File: IntegrationTests/Outer API Tests For Standards.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,765 @@
...
+							"type": "text"
+						},
+						{
+							"key": "client_secret",
+							"value": "2w6IVUoOmOLhK~SRDDz_C04oNW_s2HL-4-",
+							"type": "text"
+						},
+						{
+							"key": "resource",
+							"value": "https://citizenazuresfabisgov.onmicrosoft.com/das-{{test-environment}}-crsapi-as-ar",
```
Comment (dashton82) [inline IntegrationTests/Outer API Tests For Standards.postman_collection.json:60]: 😱

File: IntegrationTests/Outer API Tests For Standards.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,765 @@
...
+							"type": "text"
+						},
+						{
+							"key": "client_id",
+							"value": "e94608b1-0e4c-45dc-aecd-1a8442e9a58e",
+							"type": "text"
+						},
+						{
+							"key": "client_secret",
+							"value": "2w6IVUoOmOLhK~SRDDz_C04oNW_s2HL-4-",
```
Comment (dashton82) [inline IntegrationTests/Outer API Tests For Standards.postman_collection.json:55]: 😱

**Review by dashton82 — CHANGES_REQUESTED** [review]

**Comment by dashton82** [comment]
Can you contact devops about the last three please

**Review by alandburns — APPROVED** [review]

**Pull request closed without merging**